paper-server/src/main/java/org/bukkit/craftbukkit/inventory/util/CraftMenus.java

```diff
@@ -140,16 +142,16 @@ public static <V extends InventoryView, B extends InventoryViewBuilder<V>> MenuT
             return asType(new MenuTypeData<>(InventoryView.class, () -> new CraftBlockEntityInventoryViewBuilder<>(handle, Blocks.SHULKER_BOX, ShulkerBoxBlockEntity::new)));
```

Comment:
If opening noises should be prevented (as is done with chest using useFakeBlockEntity), then perhaps that could be done here too with the shulker.

Reply:
Yeah The whole shulker thing is a bit more nuanced due to the need for it to be distinctly different than a chest with GENERIC_9x3. In the interest of trying to keep this PR focused on adding default titles, this will need to be addressed in a PR where I rewrite the opening logic that Bukkit does. Long story short getNotchInventoryType isn't actually smart enough to identify the difference between a shulker box and GENERIC_9x3 as they'd be created with this API. The way around this is to define and force the use of the BlockEntity for now* The chest is okay to avoid this because it is a GENERIC_9x3.